GW2EIEvtcParser/EncounterLogic/FightLogic.cs

```diff
@@ -422,5 +423,29 @@ internal virtual void EIEvtcParse(ulong gw2Build, FightData fightData, AgentData
             ComputeFightTargets(agentData, combatData, extensions);
         }
 
+        /// <summary>
+        /// Create a <see cref="List{}"/> containing a <paramref name="buff"/> and its <paramref name="stack"/>.<br></br>
+        /// The buff must be present on any player at the end of the encounter.<br></br>
+        /// </summary>
+        /// <param name="log">The log.</param>
+        /// <param name="buff">The buff ID to add.</param>
+        /// <param name="stack">Amount of buff stacks (0-99).</param>
+        /// <returns>
+        /// A <see cref="List{T}"/> containing a <paramref name="buff"/> and its <paramref name="stack"/> if present, otherwise empty.<br></br>
+        /// To be used to add as range to <see cref="InstanceBuffs"/>.
+        /// </returns>
+        internal static List<(Buff, int)> SetOnPlayerCustomInstanceBuff(ParsedEvtcLog log, long buff, int stack = 1)
```

Comment:
Protected static would be better, this does not need to be called outside of FightLogic and its children.
Also the method would be better named "GetXXX" instead of "SetXXX" now that it returns a list instead of interactive with one.

Reply:
Should i change List to IReadOnlyList too?

Reply:
Not a Public API so not mandatory but how it is used, it would make sense